scripts/Publications_data.R

```diff
@@ -0,0 +1,56 @@
+## Script to compile all publications currently associated with the IV dataset and catalog
+## which TRNs are found in which section of each publication
+
+##############################################################################################
+
+library(tidyverse)
+library(readr)
+library(here)
+library(fs)
+library(lubridate)
+library(stringr)
+
+dir_processed <- here("data", "processed")
+
+trials <- read_csv(path(dir_processed, "trials.csv"))
+cross_registrations <- read_rds(path(dir_processed, "trn", "cross-registrations.rds"))
+
+# 'trials' gives us more information about URL and pub_type to supplement what we know from 'cross_registration'
+pubs_supplementary_info <- trials %>%
+                           select(doi, url, publication_type) %>%
+                           unique()
```

Comment:
Decided to get rid of URLs, seems to not be worth the hassle. No longer including anything from 'trials'. The resulting publications table is 437 rows instead of the original 471, but this is because of duplicated publications that only differed on URL.